docs/scos/dev/glue-api-guides/202108.0/managing-products/abstract-products/retrieving-abstract-product-prices.md

```diff
@@ -191,6 +191,7 @@ Response sample with default prices and volume prices for an abstract product
 
 </details>
 
+<a name="abstract-product-prices-response-attributes"></a>
```

Comment:
Is it normal to have a link without an HREF tag?

Reply:
that's an anchor to which I'm referring to from another place in the document